runtime/services/catalog/migrations.go

```diff
@@ -728,3 +733,20 @@ func (s *Service) updateCatalogObject(ctx context.Context, item *MigrationItem)
 
 	return catalogEntry, nil
 }
+
+// wrapMigrator is a temporary solution to log source related messages.
+func (s *Service) wrapMigrator(catalogEntry *drivers.CatalogEntry, run func() error) error {
+	if catalogEntry.Type == drivers.ObjectTypeSource {
+		s.logger.Info(fmt.Sprintf("Ingesting source %s from %s",
+			catalogEntry.Name, catalogEntry.GetSource().Properties.Fields["path"].GetStringValue()))
```

Comment:
nit: maybe use `%q` instead of `%s`?

also: does the line split look weird to you here? Wondering if a one-liner or splitting into ~4 lines would be more readable?

Reply:
I miss the auto formatting with prettier from javascript.